src/pkg/faktorySetOutcomeProcessor.go

```diff
@@ -73,7 +75,7 @@ func (s *FaktorySetOutcomeProcessor) ProcessStderr(line string) string {
 }
 
 func (s *FaktorySetOutcomeProcessor) Flush(outcome JobOutcome) {
-	vars := []opslevel.RunnerJobOutcomeVariable{}
+	vars := make([]opslevel.RunnerJobOutcomeVariable, 0)
```

Comment:
I'm just curious about this update. What is the benefit of switching to `make()`?

Reply:
GoLand suggests the same thing for me but the `var` declaration is not equivalent. [There's a bug report about this.](https://youtrack.jetbrains.com/issue/GO-5317) `make()` is considered the most idiomatic.

Reply:
fun!